@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStatus.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <UiCard>
+  <UiCard :class="hasError ? 'ui-card-backgroud-error' : 'ui-card-backgroud'">
     <UiTitle type="h4">{{ $t("status") }}</UiTitle>
-    <template v-if="isReady">
+    <template v-if="isReady && !hasError">
```

Comment:
Like we discussed earlier, I think `isReady` and `hasError` are antinomic (in my mind, if there is an error, then it's not "ready").

I propose to remove `isReady` in favor of a new `isLoading` ref which would be `true` by default.

What do you think?